SoftLayer/CLI/server/create.py

```diff
@@ -42,14 +42,9 @@
 @click.option('--postinstall', '-i', help="Post-install script to download")
 @helpers.multi_option('--key', '-k',
                       help="SSH keys to add to the root user")
-@click.option('--vlan-public',
-              help="The ID of the public VLAN on which you want the virtual "
-              "server placed",
-              type=click.INT)
-@click.option('--vlan-private',
-              help="The ID of the private VLAN on which you want the virtual "
-                   "server placed",
-              type=click.INT)
+@click.option('--no_public',
```

Comment:
use `--no-public` instead. Click automatically converts dashes to underscores. It's conventional to only have dashes for option names. Also, there are still references to vlan_public and vlan_private in the code below this.

I actually started cleaning this up before your second commit. I can take over this bit from here if you don't mind.